beesdoo_shift_swap/controllers/generate_timeslot.py

```diff
@@ -7,7 +7,7 @@
 from odoo.http import request
 
 #from odoo.addons.beesdoo_shift.models.planning import float_to_time
-
+'''
```

Comment:
To be clean this file should be removed. You can remove it in another commit. This way it will be stored in git and you can retrieve it in you need it.

Reply:
Supprimer controller